s4_reference_design.toml

```diff
@@ -64,8 +62,6 @@ atmosphere_scaling_P = 1.0
     SAT4 = { LFS=1, MFLS= 0, MFHS=0, HFS=2, site="Pole", years=7 }
 
     [telescopes.LAT]
-    # Available LAT tubes (band centers in GHz) are:
-    # ULFL (20), LFL (27,39), MFL (93,145), HFL (225,278)
     # Tubes for each LAT should be 19
 
     # Legacy survey LAT deployed in Chile
```

Comment:
@keskitalo or @smsimon can you please write here the tubes distribution for the 3 LAT?

LAT0 = { ULFL=0, LFL= 2, MFL=12, HFL=5, site="Chile", years=7 }
LAT2 = { ULFPL=1, LFPL= 2, MFPL=11, HFPL=4, site="Pole", years=7 }

Above nothing changes for the SAT, right?

Reply:
@zonca we are simulating all 85 optics tubes so there is no need to scale by optics tube counts. The two Chile LATs are identical.

Reply:
for example: 

`chile_noise_LAT_HFL1_filtered_telescope_all_time_all_bmap.fits`, I need to know how many tubes you simulated here so I can scale if someone wants a different tube distribution. same for all other tubes.

Reply:
I see. I'll get the numbers for you ASAP. SAT numbers are unchanged.

Reply:
Chile LAT (each telescope):
- 8 LF tubes
- 54 MF tubes
- 23 HF tubes
South Pole LAT (TMA):
- 4 ULF
- 9 LF
- 54 MF
- 18 HF

Reply:
@keskitalo so for SAT you simulate 1 tube at a time,
for LAT you simulate the whole amount of tubes, so for example Chile LAT MF you simulate 54 tubes.

Reply:
Correct.